integration/spark2/src/main/scala/org/apache/spark/sql/CarbonCatalystOperators.scala

```diff
@@ -101,13 +101,15 @@ case class UpdateTable(
     table: UnresolvedRelation,
     columns: List[String],
     selectStmt: String,
+    alias: Option[String],
     filer: String) extends LogicalPlan {
   override def children: Seq[LogicalPlan] = Seq.empty
   override def output: Seq[AttributeReference] = Seq.empty
 }
 
 case class DeleteRecords(
     statement: String,
+    alias: Option[String],
```

Comment:
provide a default value for optional parameter

Reply:
Ok